src/components/Query.ts

```diff
@@ -195,26 +211,29 @@ export class APIRequest {
 }
 
 function queryResponseToTableResult(body: string): TableResult {
-  return body.split("\r\n").filter(v => v !== "").reduce((table, row, index) => {
-    let fields = row.split(",").slice(3);
-
-    if (index === 0) {
-      table.Head.push(...fields);
-    } else {
-      table.Rows.push(fields);
-    }
+  return body
+    .split("\r\n")
+    .filter(v => v !== "")
+    .reduce((table, row, index) => {
```

Comment:
nit: feels worthwhile to create this reducer func outside the scope of the return, mind melt matching this up at first glance 😬